frontend/app/config/webpack.config.js

```diff
@@ -328,29 +328,14 @@ module.exports = function (webpackEnv) {
         // Support React Native Web
         // https://www.smashingmagazine.com/2016/08/a-glimpse-into-the-future-with-react-native-for-web/
         "react-native": "react-native-web",
-        "@streamlit/lib": "@streamlit/lib/src",
+        "@streamlit/lib": paths.libSrc,
```

Comment:
Have @streamlit/lib refer to the libSrc specifically rather than another library.